app/models/user.rb

```diff
@@ -12,7 +12,15 @@ class User < ApplicationRecord
 
   validates :username, presence: true
 
+  ROLES = %w[super_admin admin customer].freeze
+
   def jwt_payload
     super.merge('foo' => 'bar')
   end
```

Comment:
what does jwt mean, I will suggest we use terms everyone will easily understand.